Formula/overmind.rb

```diff
@@ -0,0 +1,21 @@
+class Overmind < Formula
+  desc "Process manager for Procfile-based applications and tmux"
+  homepage "https://github.com/DarthSim/overmind"
+  url "https://github.com/DarthSim/overmind/archive/v1.0.4.tar.gz"
+  sha256 "6a2219044fbd68757f1d1551f6750dc639b10cf9d0eab036dda25f1a0879bfe1"
+  head "https://github.com/DarthSim/overmind.git"
+
+  depends_on "go" => :build
+  depends_on "tmux"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/DarthSim/overmind").install buildpath.children
+    system "go", "build", "-o", "#{bin}/overmind", "-v", "github.com/DarthSim/overmind"
+  end
+
+  test do
+    (testpath/"Procfile").write("test: echo 'test message'")
```

Comment:
You could also add an assert above this for what happens when Procfile doesn't exist.